source/_integrations/zwave_js.markdown

```diff
@@ -974,6 +980,16 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
 
+## Z-Wave association groups used
```

Comment:
```suggestion
## Available Z-Wave association groups
```
maybe? "Used" seems not correct, as we don't use them.
However that could imply they are actually available to users, although they need another controller with a UI to configure them.

Reply:
<https://github.com/home-assistant/home-assistant.io/pull/34718>